server/index.js

```diff
@@ -1,66 +1,64 @@
-const express = require('express')
-const consola = require('consola')
-const bodyParser = require('body-parser')
-const cors = require('cors')
-const compression = require('compression')
-const helmet = require('helmet')
-const { Nuxt, Builder } = require('nuxt')
-const app = express()
-const {
-  node_env,
-  host,
-  port
-} = require('../config')
+const express = require("express");
+const consola = require("consola");
+const bodyParser = require("body-parser");
+const cors = require("cors");
+const compression = require("compression");
+// const helmet = require("helmet");
+const { Nuxt, Builder } = require("nuxt");
+const app = express();
+const { node_env, host, port } = require("../config");
 
-app.set('port', port)
+app.set("port", port);
 
 // Import and Set Nuxt.js options
-let config = require('../nuxt.config.js')
-config.dev = !(node_env === 'production')
+const config = require("../nuxt.config.js");
+config.dev = !(node_env === "production");
 
 // a security middleware that handles several kinds of attacks in the HTTP/HTTPS protocols
-app.use(helmet())
+// app.use(helmet());
```

Comment:
@bakmyradov any reason why did you comment out `helmet`?

Reply:
When dealing with package updating there was an error in the `helmet` package. I apologize I forgot to uncomment it after resolving.